app/backend/src/tests/test_auth.py

```diff
@@ -140,7 +140,7 @@ def test_invalid_token(db):
         res = api.GetUser(api_pb2.GetUserReq(user=user2.username))
 
     assert e.value.code() == grpc.StatusCode.UNAUTHENTICATED
-    assert e.value.details() == "Unauthorized"
```

Comment:
This is the last test that's failing. You should change this back to `"Unauthorized"`, instead of `errors.INVALID_TOKEN`.

This is not related to login, it's just checking that if you hit the API with the wrong token, it'll return `UNAUTHENTICATED`. If I now am on top of things, this is basically the only place that should be returning `UNAUTHENTICATED`.

Reply:
Yup, forgot to change that back, oops